docs/how-to/understanding-omnitrace-output.rst

```diff
@@ -0,0 +1,937 @@
+   "``%job%``", "Value of ``SLURM_JOB_ID`` environment variable if exists, else ``0``"
+   "``%rank%``", "Value of ``SLURM_PROCID`` environment variable if exists, else ``MPI_Comm_rank`` (or ``0`` non-mpi)"
+   "``%size%``", "``MPI_Comm_size`` or ``1`` if non-mpi"
+   "``%nid%``", "``%rank%`` if possible, otherwise ``%pid%``"
+   "``%launch_time%``", "Launch date and time (uses ``OMNITRACE_TIME_FORMAT``)"
+   "``%env{NAME}%``", "Value of environment variable ``NAME`` (i.e. ``getenv(NAME)``)"
+   "``%cfg{NAME}%``", "Value of configuration variable ``NAME`` (e.g. ``%cfg{OMNITRACE_SAMPLING_FREQ}%`` would resolve to sampling frequency)"
+   "``$env{NAME}``", "Alternative syntax to ``%env{NAME}%``"
+   "``$cfg{NAME}``", "Alternative syntax to ``%cfg{NAME}%``"
+   "``%m``", "Shorthand for ``%argt_hash%``"
+   "``%p``", "Shorthand for ``%pid%``"
+   "``%j``", "Shorthand for ``%job%``"
+   "``%r``", "Shorthand for ``%rank%``"
+   "``%s``", "Shorthand for ``%size%``"
+
+.. note::
+
+   In any output prefix key which contains a ``/`` character, the ``/`` characters
+   are replaced with ``_`` and any leading underscores are stripped. For example, 
+   an ``%arg0%`` of ``/usr/bin/foo`` translates to ``usr_bin_foo``. Additionally, any ``%arg<N>%`` keys which 
+   do not have a command line argument at position ``<N>`` are ignored.
+
+Perfetto output
+========================================
+
+Use the ``OMNITRACE_OUTPUT_FILE`` to specify a specific location. If this is an 
+absolute path, then all ``OMNITRACE_OUTPUT_PATH`` and similar
+settings are ignored. Visit `ui.perfetto.dev <https://ui.perfetto.dev>`_ and open this file.
+
+.. image:: ../data/omnitrace-perfetto.png
+   :alt: Visualization of a performance graph in Perfetto
+
+.. image:: ../data/omnitrace-rocm.png
+   :alt: Visualization of ROCm data in Perfetto
+
+.. image:: ../data/omnitrace-rocm-flow.png
+   :alt: Visualization of ROCm flow data in Perfetto
+
+.. image:: ../data/omnitrace-user-api.png
+   :alt: Visualization of ROCm API calls in Perfetto
+
+Timemory output
+========================================
+
+Use ``omnitrace-avail --components --filename`` to view the base filename for each component, as follows
+
+.. code-block:: shell
+
+   $ omnitrace-avail wall_clock -C -f
+   |---------------------------------|---------------|------------------------|
+   |            COMPONENT            |   AVAILABLE   |        FILENAME        |
+   |---------------------------------|---------------|------------------------|
+   | wall_clock                      |     true      | wall_clock             |
+   | sampling_wall_clock             |     true      | sampling_wall_clock    |
+   |---------------------------------|---------------|------------------------|
+
+With the settings ``OMNITRACE_COLLAPSE_THREADS=ON`` and ``OMNITRACE_COLLAPSE_PROCESSES=ON``, which is only valid 
+with full MPI support, the Timemory output
+combines the per-thread and/or per-rank data, which have identical call stacks.
```

Comment:
```suggestion
The settings ``OMNITRACE_COLLAPSE_THREADS`` and ``OMNITRACE_COLLAPSE_PROCESSES`` are only valid when full MPI support is enabled (link install/install.rst#mpi-support-within-omnitrace). With them set,  the Timemory combines the output of per-thread and per-rank data (respectively) of identical call stacks.
```
Something like this? I am trying to break up that long sentence.

Reply:
Sure. I'll work on it.